src/app/legacy/containers/Brand/index.jsx

```diff
@@ -16,7 +15,12 @@ const StyledBrand = styled(Brand)`
   }
 `;
 
-const BrandContainer = ({ skipLink, scriptLink, brandRef, ...props }) => {
+const BrandContainer = ({
+  skipLink = null,
+  scriptLink = null,
+  brandRef = null,
+  ...props
```

Comment:
Do we need `borderTop` and `borderBottom` here? Or is it OK since `undefined` === `false`?

```suggestion
const BrandContainer = ({
  skipLink = null,
  scriptLink = null,
  brandRef = null,
  borderTop: false,
  borderBottom: false,
  ...props
```

Reply:
They are initialised to `false` here: https://github.com/bbc/simorgh/blob/9a577693f5da22ad8a8ef391a1a588683888fb5c/src/app/legacy/psammead/psammead-brand/src/index.jsx#L167-L180